docs/userGuide/syntax/lists.md

```diff
@@ -42,6 +42,22 @@ You can use any of the [icons](../formattingContents.html#icons) supported by Ma
 If you customize any item on a certain level, you must also customize the first item on that level. If not, the list will revert to its uncustomized form.
 </box>
 
+**You can add an icon that is only for a specific item by using the `i-one-off` attribute.**
+
+<include src="codeAndOutput.md" boilerplate >
+<variable name="highlightStyle">markdown</variable>
+<variable name="code">
+* Item 1 { icon="glyphicon-education" }
+* Item 2 { icon="fas-code-branch" i-one-off="true" }
```

Comment:
```suggestion
* Item 2 { icon="fas-code-branch" i-one-off=true }
```
Let's encourage users to use a boolean if possible - the string is just to make it more robust